packages/html/src/BreakdownTable.ts

```diff
@@ -0,0 +1,214 @@
+import { HTMLTooltip } from "./HTMLTooltip";
+import { StyledTable } from "./StyledTable";
+
+export class BreakdownTable extends StyledTable {
+    protected _table;
+    protected _tbody;
+    protected _tooltip;
+    constructor() {
+        super();
+    }
+    enter(domNode, element) {
+        super.enter(domNode, element);
+        this._tooltip = new HTMLTooltip()
+            .target(domNode)
+            ;
+    }
+    update(domNode, element) {
+        const rowCount = this.useCalculatedRowCount() ? this.calculateRowCount() : this.rowCount();
+        this.transformData = function() {
+            return this.breakdownData(rowCount);
+        };
+
+        this._tooltip
+            .tooltipHTML(data => {
```

Comment:
Does this function have to be reset every update?

Reply:
It's reset on every update because the data and/or parameters may change.

Reply:
But its not accessing the data at this point in time (or any of the values as far as I can tell)?

Reply:
(It is using RowCount, but that is the only local value, which could be moved into the function?

Reply:
Valid, I'll move this to enter.